src/tv2-common/helpers/graphics/layers.ts

```diff
@@ -2,8 +2,8 @@ import { TV2BlueprintConfig } from 'tv2-common'
 import { SharedGraphicLLayer, SharedSourceLayers } from 'tv2-constants'
 
 export function GetSourceLayerForGraphic(config: TV2BlueprintConfig, name: string) {
```

Comment:
I take that it isn't everywhere where 'Graphic' or similar ones, are to be renamed, or did you just miss renaming the function name?

Reply:
Had to draw the line somewhere. Still lots of stuff called `graphic` or `grafik`...